data-imports/clean_utils.py

```diff
@@ -139,7 +139,9 @@ def clean_bbl(table, boro, block, lot):
     return sql + '''
     UPDATE {table} SET {block} = REPLACE({block}, ' ', 0);
     UPDATE {table} SET {block} = REPLACE({block}, '`', '');
+    UPDATE {table} SET {block} = preg_replace('/[^0-9]/', '', {block});
     UPDATE {table} SET {lot} = REPLACE({lot}, ' ', 0);
+    UPDATE {table} SET {lot} = preg_replace('/[^0-9]/', '', {lot});
```

Comment:
I don't have a whole lot of time but seems this issue isn't extremely time-sensitive either, so we have some leeway.  Let me look into the databases this weekend to see how easy it is to resolve.